app/Providers/RepositoryServiceProvider.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Providers;
+
+use App\Repositories\Interfaces\ProductRepositoryInterface;
+use App\Repositories\Interfaces\ServiceRepositoryInterface;
+use App\Repositories\ProductRepository;
+use App\Repositories\ServiceRepository;
+use Illuminate\Support\ServiceProvider;
+
+class RepositoryServiceProvider extends ServiceProvider
+{
+    /**
+     * Register services.
+     */
+    public function register(): void
+    {
+        $this->app->bind(ProductRepositoryInterface::class, ProductRepository::class);
```

Comment:
$this->app->bind(
ProductRepositoryInterface::class,
ProductRepository::class
);